randovania/games/am2r/json_data/The Tower.txt

```diff
@@ -153,19 +153,23 @@ Extra - map_name: rm_a4h03
   * Layers: default
   * Open Passage to Tower Exterior North East/Dock to Tower Exterior North
   > Top
-      Trivial
+      Space Jump or Walljump (Intermediate) or Can Use Spider Ball
```

Comment:
Missing morph glides, and probably CBJ too. i think with hijump you can also just cross the gap too.

Reply:
Unsure how to handle this as I thought this was just to climb the tower from below and not necessarily cross the gaps. Assuming that I just followed the already set requirements for climbing the tower (with bumped down WJ).
![image](https://github.com/randovania/randovania/assets/125271738/50e20f74-48e4-4249-b195-7b4d6c263b10)

Reply:
wrong dock. the one from west -> top is fine the one from east -> top should get changed

Reply:
Both docks to top are identical. Not sure why east requires more requirements if they are both for climbing the tower?

Reply:
![grafik](https://github.com/randovania/randovania/assets/38186597/162b0806-a1fc-4601-b276-82ca6667027b)

Reply:
I feel like that would need a second dock then? Feel weird to have a CBJ requirement if you just climbed from the left side.

Reply:
maybe. ~~the way it's currently setup, is that the room below has the requirements to reach the right side here, thus the connection from this dock to top being marked as trivial.~~ nevermind, apparently this is not the case.
leave it as is then, and add a todo into the comments and we'll look at that one later.